src/auth.js

```diff
@@ -56,6 +57,9 @@ const nullAdapter = require('./adapters/auth/null.js');
  * @property {string[]} [denyUsers]
  */
 
+/** Keeps track of the active refresh tokens. */
+let refreshTokens = [];
```

Comment:
This makes the web server stateful.

I think we should look into some sort of simple database store, just as it was in the cookie implementation.

If not, scaling will basically not work since instances are unaware of each others tokens.

Reply:
Do you have any suggestions for a package/mechanism to use for the database store?

Reply:
The "classic" session stuff used loki as an in-memory store with the ability to dump to disk for persistence. This could be swapped out with technically anything in the config, like Redis so that you could share stuff between instances (scaling).

So maybe just a basic in-memory store to start with ? That way we don't put the state in the web server code, but have it floating on the side. It's just a matter of basic CRUD operations here, so technically anything will do and can be written in such a way that we can make it swappable.